api/V1/Device.js

```diff
@@ -89,6 +89,59 @@ module.exports = function(app, baseUrl) {
     })
   );
 
+  /**
+  * @api {get} /api/v1/devices/users Get all users who can access a device.
+  * @apiName GetDeviceUsers
+  * @apiGroup Device
+  * @apiDescription Returns all users that have access to the device
+  * through both group membership and direct assignment.
+  *
+  * @apiUse V1UserAuthorizationHeader
+  *
+  * @apiParam {Number} deviceId ID of the device.
+  *
+  * @apiUse V1ResponseSuccess
+  * @apiSuccess {JSON} rows Array of users who have access to the
+  * device. Each element includes `id` (user id), `username`, `email`,
+  * `relation` (either `group` or `device`) and `admin` (boolean).
+  * @apiUse V1ResponseError
+  */
+  app.get(
+    apiUrl + '/users',
+    [
+      middleware.authenticateUser,
+      middleware.getDeviceById(body),
```

Comment:
You will want to use `query` here instead of `body` for a GET request.
The middleware looks to load the device object into the body though.

Reply:
Good catch! The test code is making the request incorrectly too so I didn't notice.

Serves me right for copying and pasting.